python/triton/compiler/config.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+
+def front_end_debugging():
+    return os.getenv("TRITON_FRONT_END_DEBUGGING", "0") == "1"
```

Comment:
@peterbell10 can you document it in readme?

Reply:
Yes please, I looked for this myself at one point and ended up patching the code similarly to how it was done here. I am sure it would be useful to others :)